### DISCUSSION TITLE
Vercel postgres "Critical dependency: require function is used in a way in which dependencies cannot be statically extracted" during local dev

### DISCUSSION THREAD

**Discussion by VoloDziu** [vercel/community#4914] [Help]
### Summary

Hi folks,

Trying out the most basic case of using vercel's postgres to load data in a nextjs server component, but during local development it keeps throwing me warnings in the console.

```
Critical dependency: require function is used in a way in which dependencies cannot be statically extracted

Import trace for requested module:
./node_modules/node-gyp-build/index.js
./node_modules/bufferutil/index.js
./node_modules/ws/lib/buffer-util.js
./node_modules/ws/lib/receiver.js
./node_modules/ws/wrapper.mjs
./node_modules/@vercel/postgres/dist/index-node.js
./src/app/page.tsx
 ⚠ ./node_modules/node-gyp-build/index.js
```

Everything seems to work correctly, but I am curious if I am missing something or if these warnings are expected during local dev?

### Example

https://github.com/VoloDziu/next-form

### Steps to Reproduce

- Create a new next app with `@vercel/postgres` 
- link it to a vercel project with storage - a single table with 3 columns: name (string), email (string), id(number)
- change the `page.tsx` to the following

```ts
//     "@vercel/postgres": "^0.5.1",
//    "next": "14.0.3"

import { sql } from "@vercel/postgres";

export default async function Home() {
  const { rows } = await sql`SELECT * FROM submissions`;

  return (
    <main className="flex min-h-screen flex-col items-center justify-center p-24">
      {rows.map((submission) => (
        <div key={submission.id} className="flex gap-2">
          {submission.name} {submission.email}
        </div>
      ))}
    </main>
  );
}
```

**Comment by Renderthegreat** [comment]
I'm having the same issue for now try to import a specific version if possible by adding @version to @vercel/postgres
